packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/WfoTableWithFilter.tsx

```diff
@@ -64,7 +65,7 @@ export type WfoTableWithFilterProps<T extends object> = {
     onUpdateQueryString: (queryString: string) => void;
     onUpdatePage: (criterion: Criteria<T>['page']) => void;
     onUpdateDataSort: (dataSorting: WfoDataSorting<T>) => void;
-    hasError?: boolean;
+    error?: RTKQueryError;
```

Comment:
There can be errors that are not RTKQueryErrors I think so this typing isn't always correct right?

DISCUSS

Reply:
fixed with the mapper